originals/t.yaml

```
@@ -1,3 +1,23 @@
+- name: Taiko no Tatsujin
+  external:
+    wikipedia: Taiko no Tatsujin
+  platform:
+  - Arcade
+  meta:
+    genre:
+    - Rhythm
+
+- name: Team Fortress 2
+  names:
+  - TF2
+  external:
+    wikipedia: Team Fortress 2
+  platform:
+  - Windows
```

```suggestion
  - Linux
  - OSX
  - PlayStation 3
  - Windows
  - Xbox 360
```
Add PlayStation 3 and Xbox 360 to schema